src/Benchmarks.ServerJob/ServerJob.cs

```diff
@@ -43,5 +43,8 @@ public IEnumerable<Source> Sources
 
         [JsonConverter(typeof(StringEnumConverter))]
         public WebHost WebHost { get; set; }
+
+        [JsonConverter(typeof(StringEnumConverter))]
+        public TargetFramework TargetFramework { get; set; }
```

Comment:
I would rename `TargetFramework` to just `Framework`, since it's shorter and it matches the command-line parameter `--framework` .